app/web/features/auth/phone/ChangePhone.tsx

```diff
@@ -168,25 +158,15 @@ export default function ChangePhone({
             <Typography variant="body1">
               {t("auth:change_phone.no_phone_description")}
             </Typography>
-            <Controller
+            <PhoneInputWithCountry
               name="phone"
               control={control}
               rules={{
-                validate: (value) => isValidPhoneNumber(value),
+                validate: (value: string) => isValidPhoneNumber(value),
               }}
-              render={({ field }) => (
-                <ForwardedRefPhoneInput
-                  {...field}
-                  name="phone"
-                  control={control}
-                  rules={{
-                    validate: (value) => isValidPhoneNumber(value),
-                  }}
-                  international
-                  placeholder={t("auth:change_phone.phone_label")}
-                  id="phone"
-                />
-              )}
+              international
+              placeholder={t("auth:change_phone.phone_label")}
+              id="phone"
             />
```

Comment:
In case anyone asks me why I did this:

https://gitlab.com/catamphetamine/react-phone-number-input#react-hook-form

Then it handles forwarding the ref for react-hook-form without me writing tons of code to do it.